2_munge/src/1_PlotPrep_and_Summary.R

```diff
@@ -0,0 +1,59 @@
+##This script munges our Mendota data to prep it for Figure 1 and then
+##outputs a text summary of the results
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(whisker)
+library(readr)
+
+## Function for plotting the data and comparing models
+munge_data <- function(fileIn, project_output_dir, name){
```

Comment:
We have a team convention of using `camel_case` for variable and function names. Can you change `fileIn`?

Reply:
Also, `name` could be more specific - can you choose a name that gives a stronger hint about how the argument will be used?